dev-docs/source/GettingStarted.rst

```diff
@@ -90,6 +90,61 @@ where ``X.Y.Z`` should be replaced with the version that was downloaded.
 
 if you wish to setup eclipse for use developing mantid, then instructions can be found :ref:`here <Eclipse>`.
 
+Ubuntu 20.04
+~~~~~~~~~~~~
+- Mantid is not yet officially supported on Ubuntu 20.04 as Qt4 has been removed but Workench can be built by installing:
```

Comment:
```suggestion
- Mantid is not yet officially supported on Ubuntu 20.04 as Qt4 has been removed but Workbench can be built by installing:
```

Reply:
Sorry, great to have these instructions, but I think it's worth correcting this typo.

Reply:
Oh yes absolutely! Thanks!